cpuid/src/cpu_leaf.rs

```diff
@@ -43,18 +55,18 @@ pub mod leaf_0x1 {
         pub const PDCM_SHIFT: u32 = 15;
         // 18 = DCA Direct Cache Access (prefetch data from a memory mapped device)
         pub const MOVBE_SHIFT: u32 = 22;
-        pub const TSC_DEADLINE_TIMER_SHIFT: u32 = 24;
+        pub const TSC_DEADLINE_TIMER_BITINDEX: u32 = 24;
```

Comment:
Looks like now we have two different suffices for the same thing (i.e. `BITINDEX` and `SHIFT`). Looks a bit confusing.